Digipost/APIClient.swift

```diff
@@ -360,26 +339,26 @@ class APIClient : NSObject, NSURLSessionTaskDelegate, NSURLSessionDelegate, NSUR
                 self.uploadProgress?.completedUnitCount = totalSent
             })
         }
+        validateFullScope { () -> Void in
+            let task = self.fileTransferSessionManager.dataTaskWithRequest(urlRequest, completionHandler: { (response, anyObject, error) -> Void in
+                dispatch_async(dispatch_get_main_queue(), {
+                    self.removeTemporaryUploadFiles()
+                    self.isUploadingFile = false
+                    if self.isUnauthorized(response as! NSHTTPURLResponse?) {
```

Comment:
commit: d09355c5537deaa377419fb7b54b4652c6c5a765 
Added a really short timeout so its easy to test refresh of access tokens
and removing test values for timeout in this commit after: 3ee2798fd9ac3721cd776ea138424c5163122319